ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -242,3 +242,39 @@ def test_paddle_rfftfreq(
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="paddle.fft.irfftn",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("complex"),
+        min_value=-10,
+        max_value=10,
+        min_num_dims=1,
+        max_num_dims=4,
+        min_dim_size=2,
+        valid_axis=True,
+        force_int_axis=True,
+    ),
+    norm=st.sampled_from(["backward"]),
+)
+def test_paddle_irfftn(
+    dtype_x_axis,
+    norm,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        x=x[0],
+        s=None,
```

Comment:
s=none, otherwise the input people should determine the shape of output exactly macthing the result which is an extra step, if input is wrong, then the result is wrong.

a nomarl step taken by tensorflow paddle and numpy is to calculate the shape of x to determine the normal axes and last axis, they similar use this stragety to handle when the s is None, so s is calcualted from the function.